settings.sample.ini

```diff
@@ -0,0 +1,49 @@
+; This is a sample settings file. Copy it to "settings.ini" and configure it
+; before running make.
+
+; To load variables in bash use:
+; > source $current_dir/common.sh
+; > declare -A LIST_OF_APPS
+; > get_settings_section settings.ini fdroid_apps LIST_OF_APPS
+
+
+; General settings.
+[general]
+system_app_destination=/system/priv-app ; no trailing slash.
+user_app_destination=/data/app ; no trailing slash.
+;password_file=/path/to/secret/file
+
+; Preferred snapshot version
+snapshot_version=M12
+
+
+; Installed before those downloaded from the f-droid.org repository.
+[apps]
+FDroid=https://f-droid.org/FDroid.apk
+OrfoxFennec-debug=https://guardianproject.info/builds/OrfoxFennec/latest/OrfoxFennec-debug.apk
+
+
+; Contains a list of applications to be installed.
+; If you feel adventurous use "latest" to retrieve the latest released version.
+; NOTE: Versions are locked because we want to be sure that the apps are known
+;       to be secure.
+[fdroid_apps]
+com.artifex.mupdfdemo=55
+com.csipsimple=2459
+com.fsck.k9=22002
+com.morlunk.mumbleclient=72
+com.projectsexception.myapplist.open=16
+com.xabber.androiddev=123
+de.schildbach.wallet=209
+fr.kwiatkowskiTrack=1
```

Comment:
Fixed, and i also updated the version code, code 1 is no longer in the F-Droid repository.